app/ui.js

```diff
@@ -13,7 +13,7 @@
 
 import * as Log from '../core/util/logging.js';
 import _, { l10n } from '../core/util/localization.js';
-import { isTouchDevice, browserSupportsCursorURIs as cursorURIsSupported } from '../core/util/browsers.js';
+import { isTouchDevice } from '../core/util/browsers.js';
 import { setCapture, getPointerEvent } from '../core/util/events.js';
 import KeyTable from "../core/input/keysym.js";
 import keysyms from "../core/input/keysymdef.js";
```

Comment:
make sure all of these imports are still needed.

Reply:
`Base64` is the only one not used. But it's not used on `master` either, so feel free to remove it there.

`KeyTable` is only used by `sendCtrlAltDel()`, which I considered removing since it is just a wrapper around `sendKey()`. Does anyone of you want to keep it?

Reply:
I'm fine with removing sendCtrlAltDel

Reply:
`Display` doesn't seem to be used either.

Reply:
Ehm? It's instantiated in the RFB constructor. :)